template.nunjucks

```diff
@@ -151,7 +151,7 @@ FORMAT: 1A
 ## Data Structures
 
 {% for dataStructure in dataStructures.content %}
-{{ dataStructure.content|mson }}
+{{ dataStructure|dataStructure }}
```

Comment:
Resource data structures (example: https://github.com/apiaryio/fury-adapter-apib-serializer/blob/kylef/update-fury/test/fixtures/sample.json#L97) also contain an `id` so the existing code saw `id` on dataStructure element and assumed it was a standalone `## Data Structure` and would render it as such incorrectly.

I split out the existing code into two separate filters (one still called mson) and another called `dataStructure` as I felt this was a better name in the content of rendering a data structure. This allows `dataStructure` filter to use `##` headings and the other `mson` (attributes) which render the attributes as list items.